aim/db/models.py

```diff
@@ -666,3 +674,221 @@ class Topology(model_base.Base, model_base.AttributeMixin,
     __tablename__ = 'aim_topologies'
     __table_args__ = (model_base.uniq_column(__tablename__, 'name') +
                       model_base.to_tuple(model_base.Base.__table_args__))
+
+
+class VMMController(model_base.Base, model_base.HasDisplayName,
+                    model_base.HasAimId, model_base.AttributeMixin,
+                    model_base.IsMonitored, model_base.HasName):
+    """DB model for VMM Controller."""
+    __tablename__ = 'aim_vmm_controllers'
+    __table_args__ = (
+        model_base.uniq_column(__tablename__, 'domain_type', 'domain_name',
+                               'name') +
+        (sa.ForeignKeyConstraint(
```

Comment:
With the AIM-as-a-Schema approach, we should be able to create objects independently of their ACI parents, making the use of FKs an inconsistency between SQL and K8S. Even from a testing perspective, we want the same test suite to be able to run on both systems, which means that we eventually want to remove all FKs. For now, we might think about not adding new ones.